openpype/hosts/fusion/deploy_ayon/fusion_shared.prefs

```diff
@@ -0,0 +1,19 @@
+{
+Locked = true,
+Global = {
+  Paths = {
+    Map = {
+      ["AYON:"] = "$(AYON_FUSION)/deploy_ayon",
```

Comment:
Due to _this_ referring to an env var I feel like we can - reuse the exact same logic in OpenPype and AYON but only change the env var itself. Similarly `AYON:Config` could become: `AYON:$(AYON_OR_OPENPYPE_DEPLOY_NAME)` or alike so it picks based on whatever env var we'd set? Just so this file needs to duplicating.

Reply:
see current changes here 120c0d0b608f684af947cb8d30781cd8c458f27d